c++/src/kj/thread.c++

```diff
@@ -42,7 +44,7 @@ Thread::~Thread() noexcept(false) {
   if (!detached) {
     KJ_ASSERT(WaitForSingleObject(threadHandle, INFINITE) != WAIT_FAILED);
 
-    KJ_IF_MAYBE(e, exception) {
+    KJ_IF_MAYBE(e, state->exception) {
```

Comment:
Hmm, it looks like I totally forgot to make the updates to win32 ~Thread that I did to pthread ~Thread.